packages/office-ui-fabric-react/src/components/TextField/examples/TextField.AutoComplete.Example.tsx

```diff
@@ -1,17 +1,23 @@
 import * as React from 'react';
 import { PrimaryButton } from 'office-ui-fabric-react/lib/Button';
 import { TextField } from 'office-ui-fabric-react/lib/TextField';
+import { getId } from 'office-ui-fabric-react/lib/Utilities';
 import './TextField.Examples.scss';
 
 export class TextFieldAutoCompleteExample extends React.Component<any, any> {
+  private _textFieldId: string = getId('autoCompleteTextField');
+  private _labelId: string = getId('autoCompleteLabel');
+
   public render(): JSX.Element {
     return (
-      <form action="" className="docs-TextFieldExample">
-        <TextField
-          label="Fill in and submit this form. The page will reload and autocomplete suggestions will appear."
-          name="example"
-          autoComplete="on"
-        />
+      <form action="" className="docs-TextFieldExample" aria-labelledby={this._labelId}>
+        <div className="ms-TextFieldStandaloneLabel">
+          <label htmlFor={this._textFieldId} id={this._labelId}>
+            Fill in and submit this form. The page will reload and autocomplete suggestions will appear.
+          </label>
+        </div>
+        <TextField id={this._textFieldId} aria-describedby={this._labelId} label="" name="example" autoComplete="on" />
```

Comment:
Don’t think you should need the aria-describedby if the label’s for is set up properly. Also, the label probably doesn’t need an id. (And come to think of it I don’t understand the need for a separate label rather than a label property?)

Reply:
It looks like if we use `TextField`'s `label` attribute, the component will render a label with the `htmlFor` automatically populated with the correct ID, see https://github.com/OfficeDev/office-ui-fabric-react/blob/78a7e9b198b18e4804c77b7bfc8b3ec96fad76e2/packages/office-ui-fabric-react/src/components/TextField/TextField.base.tsx#L379

Reply:
The gymnastics here are to give a description to the `form` element. unfortunately we do not have the capability to pull out a label's `id` in order to hand _that_ to the form's aria props.

Reply:
Interesting...stepping back a bit, do you know why this example in particular uses a `<form>` when none of the others do? Could we just take that out? (Or if you want to keep it, some comments explaining the reason for the separate label would be good.)

Reply:
This example uses the form to trigger the browser to understand that the form has been submitted and thus enable autocomplete.

Reply:
Added a comment to explain. I'm not convinced this is the most useful example in the world. Happy to remove entirely if @ecraig12345 would like.